travis/before_install.sh

```diff
@@ -2,19 +2,47 @@
 
 # Don't source setup.sh here, because the virtualenv might not be set up yet
 
+export NUMCORES=`grep -c ^processor /proc/cpuinfo`
+if [ ! -n "$NUMCORES" ]; then
+  export NUMCORES=`sysctl -n hw.ncpu`
+fi
+echo Using $NUMCORES cores
+
 # Install dependencies
-brew install ccache protobuf
-
-# Install Python.
-# The brew update/upgrade commands are needed because otherwise homebrew
-# exits with an error code when installing python2 since it is already installed
-# but with a different version.
-if [ "${PYTHON_VERSION}" != "python2" ]; then
-  brew install ${PYTHON_VERSION}
+if [ "$TRAVIS_OS_NAME" == "linux" ]; then
+  # Install GCC 5
```

Comment:
nit: you probably don't need gcc5. the reason onnx-fb-universe needs it is pytorch had a bug before that can not be built with gcc4